openreview/arr/process/comment_process.py

```diff
@@ -0,0 +1,182 @@
+def process(client, edit, invitation):
+
+    domain = client.get_group(edit.domain)
+    venue_id = domain.id
+    meta_invitation_id = domain.get_content_value('meta_invitation_id')
+    short_name = domain.get_content_value('subtitle')
+    contact = domain.get_content_value('contact')
+    authors_name = domain.get_content_value('authors_name')
+    submission_name = domain.get_content_value('submission_name')
+    reviewers_name = domain.get_content_value('reviewers_name')
+    reviewers_anon_name = domain.get_content_value('reviewers_anon_name')
+    reviewers_submitted_name = domain.get_content_value('reviewers_submitted_name')
+    sender = domain.get_content_value('message_sender')
+    comment_threshold = domain.get_content_value('comment_notification_threshold')
+
+    submission = client.get_note(edit.note.forum, details='replies')
+    comment = client.get_note(edit.note.id)
+    paper_group_id=f'{venue_id}/{submission_name}{submission.number}'
+
+    ### TODO: Fix this, we should notify the use when the review is updated
+    if comment.tcdate != comment.tmdate:
+        return    
+
+    ignore_groups = comment.nonreaders if comment.nonreaders else []
+    ignore_groups.append(edit.tauthor)
+
+    signature = comment.signatures[0].split('/')[-1]
+    pretty_signature = openreview.tools.pretty_id(signature)
+    pretty_signature = 'An author' if pretty_signature == 'Authors' else pretty_signature
+
+    # Count comments between reviewer and authors
+    comment_count = 0
+    signed_by_reviewer = 'Reviewer' in signature
+    signed_by_author = 'Authors' in signature
+
+    for reply in submission.details['replies']:
```

Comment:
So here you are checking all replies to the submission, not just comments within a thread?
So after a while, no more emails will be sent to PC, SACs and ACs at all, even if a new thread is started?